### DISCUSSION TITLE
Better performance with Erlang multisend?

### DISCUSSION THREAD

**Discussion by benbro** [processone/ejabberd#4321] [Ideas]
There is a [discussion](https://erlangforums.com/t/send-optimization-for-sending-to-many-processes/2200/16) and [PR](https://github.com/erlang/otp/pull/8072) for multisend support - being able to send the same message to many pids. Might be relevant to Ejabbered for broadcasting the same message to many clients in a large chat room.

**Comment by prefiks** [comment]
Yes, this could potentially help, but we really don't send that much identical messages. Each message that we send has to have recipient jid in it, and that makes each message different. But maybe it would be possible to do substitution on receiver end? I will take a look at that.
